### PR TITLE
Feature/go stdlib json

### PR DESCRIPTION
**Pull request by seilis**
- Add Python serialization test.
- Add Golang JSON serialization/deserialization test based on the standard library.

### CODE REVIEW

File: tests/go-stdlib-json.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    small_duration, total_small_duration = dump_json(small, 786217)
+    fmt.Printf("go-stdlib-json-serialize,%v,%v,%v,%v\n", small_duration,
+               total_small_duration, big_duration, total_big_duration)
+}
+
+
+func load_json(jFile string, iterations int) (float64, float64) {
+    jsonBytes, err := ioutil.ReadFile(jFile)
+    if err != nil {
+        fmt.Println("Error! Could not open file ", jFile)
```
Comment (tvieregge) [inline tests/go-stdlib-json.go]: Return here?

File: tests/go-stdlib-json.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    // Amount of time it took to load the JSON file
+    diff := end.Sub(start)
+    return 1e-9*float64(diff.Nanoseconds())/float64(iterations), 1e-9*float64(diff.Nanoseconds())
+}
+
+
+func dump_json(jFile string, iterations int) (float64, float64) {
+    jsonBytes, err := ioutil.ReadFile(jFile)
+    if err != nil {
+        fmt.Println("Error! Colud not open file ", jFile)
```
Comment (tvieregge) [inline tests/go-stdlib-json.go]: Return here?

File: tests/go-stdlib-json.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+func load_json(jFile string, iterations int) (float64, float64) {
+    jsonBytes, err := ioutil.ReadFile(jFile)
+    if err != nil {
+        fmt.Println("Error! Could not open file ", jFile)
+    }
+
+    start := time.Now()
+    // Output data
+    var data map[string]interface{}
```
Comment (tvieregge) [inline tests/go-stdlib-json.go]: This doesn't get timed in other cases. Probably not a noticeable amount of time, but symmetry...

**Comment by seilis** [comment]
Comments fixed.

**Pull request merged**